### ISSUE TITLE
hook up  download() of WARCs ... into correct dir? ??

### ISSUE THREAD

**Issue by ndushay** [sul-dlss/wasapi-downloader#31]
once we have the WasapiConn class, we should be able to implement a download() method for ware files (which would be called from method downloading crawl).

We will probably need an output directory from the settings for this ...  (currently I envision `[crawl id]` appended to the output dir from settings)

probably in WasapiCrawlSelector OR in WasapiDownloader class.

Assumption:  if there is more than one location given, we will try the first location.   Creating a separate ticket for multiple locations. (#34)

NOTE:  doing the checksum validation is a separate ticket.  However, writing to (settings basedir)/(crawlid)/warc  as the location should come along for free (perhaps not a ticket, but a goal in the milestone)

**Comment by ndushay** [comment]
blocked by #38 and #20
